ts/components/wallet/paymentSummary/PaymentSummaryComponent.tsx

```diff
@@ -42,22 +37,87 @@ const styles = StyleSheet.create({
   toAlignColumnstart: {
     flexDirection: "column",
     alignItems: "flex-start"
+  },
+
+  underlined: {
+    textDecorationLine: "underline",
+    color: variables.brandLight
+  },
+
+  iconMargin: {
+    marginBottom: 3,
+    marginLeft: 3
   }
 });
 
 export default class PaymentSummaryComponent extends React.Component<Props> {
+  /**
+   * Depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component. If the values differ, then the user can display both the values
+   * and a brief exmplanation related to the update.
+   */
+  private isAmountUpdated() {
+    return this.props.updatedAmount !== this.props.amount;
```

Comment:
we need to pay special attention to this since we're comparing two strings, perhaps the amount is the same but the strings differ (e.g. `10,00` vs `10,0`) ? how can we make sure we're correctly comparing the amounts? - @fgiobergia

Reply:
the component is provided by 2 numbers coming from the API. Anyway, we created a story which will imply a review and implementation of further controls on the fields.